Libraries/PeriphDrivers/Source/DMA/dma_reva.c

```diff
@@ -500,7 +500,11 @@ int MXC_DMA_RevA_MemCpy(mxc_dma_reva_regs_t *dma, void *dest, void *src, int len
         return retval;
     }
 
+#if (TARGET_NUM == 32657)
```

Comment:
:(

Reply:
Yep, agreed. I don't like this either lol. We can change this in another separate PR.